src/features/profile/profile_section.ts

```diff
@@ -39,7 +40,8 @@ export function getProfileSection(options: ProfileOptions): AnyHomeTabBlock[] {
           text:
             `*👤 Status* · ${options.status}\n` +
             `*⭐️ Community Credits* · ${options.communityCredits}/6\n` +
-            "*🏆 Liga* · Credit Warrior",
+            `*🥇 Rank* · ${options.rank}\n` +
+            `*🏆 Liga* · Credit Warrior`,
```

Comment:
Lets remove the liga, since its static anyways and would break the layout now anyways.